package.json

```diff
@@ -69,11 +69,16 @@
       "@oclif/plugin-help"
     ],
     "topics": {
-      "limits": {
-        "description": "display current org’s limits",
+      "force": {
+        "external": true,
```

Comment:
what does external:true do?

Reply:
I don't know, I just copied the json snippet from other plugins where the CLi ref is working.  @amphro ?

Reply:
this is correct.  Thomas explained it for me.

Reply:
So what does it do?

Reply:
It says that the description is defined in another plugin so we don't have multiple plugins defining the same description. Maybe it should be renamed. It is only a property used by the command reference generator.